### PR TITLE
feat(spellcheck): spellcheck headline/abstract

### PR DESCRIPTION
**Pull request by petrjasek**
switch those fields to use sd-text-editor directive with spellchecker

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/article-edit.html [hunk 1]
```diff
@@ -10,33 +10,41 @@
...
+    <div id="title" class="headline"
+        sd-text-editor
+        data-config="{disableToolbar: true, disableReturn: true}"
         ng-model="item.headline"
         ng-change="autosave(item)"
         ng-if="_editable"
-        ng-trim="false"></textarea>
-	<div class="headline" ng-if="!_editable">{{item.headline}}</div>
+        ng-trim="false"></div>
+	<div class="headlbine" ng-if="!_editable">{{item.headline}}</div>
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/views/article-edit.html]: spelling error headline instead headlbine

Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/views/article-edit.html]: right thx

**Comment by amagdas** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**